client/my-sites/checkout/utils.ts

```diff
@@ -135,3 +135,24 @@ export function getAffiliateCouponLabel(): string {
 	// translators: The label of the coupon line item in checkout
 	return translate( 'Exclusive Offer Applied' );
 }
+
+export function getCouponLabel(
```

Comment:
Do you plan to keep these new methods in production after the experiment is finished? If so, then you may want to abstract the experiment code into a custom hook rather than embed it in the function.

Reply:
Yes the methods will be in production after the experiment is finished. I was planning on removing the experiment references once the experiment is completed. Went with this route since it will be removed.